app/server.js

```diff
@@ -23,6 +23,25 @@ app
       }
     });
 
+    server.get("/editprofile", (req, res) => {
```

Comment:
We use the following URL for this:

```suggestion
    server.get("/user/<userid>/profile", (req, res) => {
```

When building this, think about what happens when a user edits a profile of another user.

Reply:
I will fix this on a different branch as it is part of [RFE-166](https://getopensocial.atlassian.net/jira/software/projects/RFE/boards/109?selectedIssue=RFE-166).